package.json

```diff
@@ -38,7 +38,7 @@
   },
   "devDependencies": {
     "@netlify/build": "^27.4.0",
-    "@netlify/eslint-config-node": "6.0.0",
+    "@netlify/eslint-config-node": "5.1.8",
```

Comment:
downgraded to avoid lint errors with module resolution after syncing package.json and package-lock.json

Reply:
Odd that this is happening 🤔 Is it the rules within this project that are causing these issues?

Reply:
Are the errors related to the files you've changed, or elsewhere? If they're not related to your changes then it seems like this shouldbe a different PR: i.e. you do a separate PR to update the package-lock and if that breaks the linting, make the linting changes in there rather than in this unrelated PR